homeassistant/components/dyson/sensor.py

```diff
@@ -195,5 +195,5 @@ def __init__(self, device):
     def state(self):
         """Return Air Quality value."""
         if self._device.environmental_state:
-            return int(self._device.environmental_state.volatil_organic_compounds)
```

Comment:
This typo is also in the base lib https://github.com/etheralm/libpurecool/blob/3d8eab13fa9e9d67d215f67d1915bbd88f040527/libpurecool/dyson_pure_state.py#L132

Reply:
yes. it appeared in both ways in the tests and apparently it does so also in the library. definitely looks like a library bug since it should be the same interface for v1 and v2

Reply:
in any case, it looks like the library maintainer and the component is the same, so he/she can probably shed some more light here